ids/mapping.json

```diff
@@ -15,6 +15,7 @@
   "khanjan": "Khanjan04",
   "kratika": "Kratika0907",
   "manjot": "manjots1607",
+  "moses": "moses-kamara",
```

Comment:
As per mentioned in README.md this is the expected format - `[member_rds_id]: "<github_id>"`. So, please replace `moses-kamara` with your github id. 

For more information refer : https://github.com/Real-Dev-Squad/website-static/blob/main/README.md#ids

Reply:
Ok. Will get this done today.